docs/source/fundamentals/interfaces.rst

```diff
@@ -37,6 +37,18 @@ executed at the top of your session. Please note:
 
 - This loads the extension to the current session only, i.e., it does not carry over to different sessions; you will need to repeat it for each new session.
 
+.. note::
+
+    LineaPy works with JupyterHub!
+
+    If LineaPy is installed by the admin, it will be accessible to every user and the admin can set the LineaPy 
+    extension to be automatically loaded by adding ``c.InteractiveShellApp.extensions = ["lineapy"]`` in 
+    `ipython_config.py <https://ipython.readthedocs.io/en/stable/config/intro.html>`_.
+
+    If LineaPy is installed by an individual user during their session, it will be accessible to that particular
+    user only as long as they do not have a write permission on the shared environment.
+    In this case, the user will need to run ``%load_ext lineapy`` as explained above.
```

Comment:
@mingjerli I moved Jupyter Hub instructions to a more relevant section. Please let me know if you have questions about this change.

Reply:
@yoonspark I think it make more sense to keep it in setup section. The person who configures JupyterHub is more like a administrator not a user.

Reply:
Okay, moving it back to that section.